src/screens/Customers/AddCustomer.tsx

```diff
@@ -184,31 +205,32 @@ function AddCustomer(props: AddCustomerProps) {
           <div className={classes.textContainer}>
             <TextField
               label={'Meter Number'}
-              id={'meter-number'}
+              id={'meterNumber'}
               placeholder={'Input'}
```

Comment:
Not related to this PR but this probably isn't the intended placeholder?

Reply:
I'll change it in this PR to `e.g. <some number>` since that seems to be how we're doing it.